gammapy/data/event_list.py

```diff
@@ -377,7 +374,9 @@ def plot_energy_hist(self, ax=None, ebounds=None, **kwargs):
             ebounds = EnergyBounds.equal_log_spacing(emin, emax, 100)
 
         spec = CountsSpectrum(energy_lo=ebounds[:-1], energy_hi=ebounds[1:])
-        spec.fill(self.energy)  # leaving spec.fill(self) was triggering an issue for the LAT event list
+        spec.fill(
+            self.energy
+        )  # leaving spec.fill(self) was triggering an issue for the LAT event list
```

Comment:
This is an example where the formatting isn't great and manual formatting would improve.
If the comment is moved on the line before, the code will fit on one line.